client/my-sites/checkout/checkout-thank-you-i2/masterbar/masterbar-styled.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
```

Comment:
Just copying over the component from Marketplace, maybe we should clean it up or maybe not?